### PR TITLE
Creating a Dyno RPG Mini Game

### PR DESCRIPTION
**Pull request by ghost**
- Create infect.txt
- Create plague.txt
- Create curse.txt
- Create blight.txt
- Create fester.txt
- Create sacrifice.txt
- Create treat.txt
- Create cure.txt
- Create divinecure.txt
- Create stab.txt
- Create dethmatch.txt
- Create README.md

### CODE REVIEW

File: Fun Commands/Mini-Games/Dyno-RPG/sacrifice.txt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{delete}
+{choose: Cthulu; The Flying Spaghetti Monster; the Illuminati; Ben Brode; a local school district; a giant squid; Herobrine; the devil; RNGesus; Joe Bob; get the iPhone X; Santa Claus; God-Emperor Trump; cure cancer; create a new meme; Nyarlathotep; Yogg-Saron; N'Zoth; Y'Shaarj; C'Thun; McDonalds; Dictator Advaith; Slenderman}
```
Comment (Swordington) [inline Fun Commands/Mini-Games/Dyno-RPG/sacrifice.txt:2]: Dictator Advaith nice

File: Fun Commands/Mini-Games/Dyno-RPG/treat.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{delete}
+{silent}
+{!role $1 -Infected}
```
Comment (Swordington) [inline Fun Commands/Mini-Games/Dyno-RPG/treat.txt:3]: Is this a command what you want to ensure can only be used by someone who isn't infected?

File: Fun Commands/Mini-Games/Dyno-RPG/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+For this to work you will need the following roles in your guild
+
+- `Festered`
+- `Blighted`
+- `Bleeding`
+- `Infected`
+- `Cursed`
+
+
+### *This is a ongoing project and will have more commands added over time. Anyone who wishes to contribute their own commands would of course be much appreciated*
```
Comment (Swordington) [inline Fun Commands/Mini-Games/Dyno-RPG/README.md:28]: You may want to add a self-credit in the footer

**Review by Swordington — COMMENTED** [review]
Overall, the mini-game looks very nice! However, it seems that it's possible for even someone who isn't infected to infect (and continuing throughout the different actions) which may be intentional game design or may be an oversight.

File: Fun Commands/Mini-Games/Dyno-RPG/sacrifice.txt (hunk 1 above)
Comment (advaith1) [reply Fun Commands/Mini-Games/Dyno-RPG/sacrifice.txt:2]: 👀

**Comment by Swordington** [comment]
@PhantomNimbi If you're still working on this PR please take a look at the comments left, if not we'll close it in a few days.

**Comment by Swordington** [comment]
Closing due to inactivity

**Comment by ghost** [comment]
Yo completely forgot about this one. And yes I'lll go ahead and get back onto this one lmao. I will need to reinv dyno as i swapped to a yagpdb to self host and eventually got peeved at them and just dropped them entirely lol. Give me a bit and I'll get started immediately

**Pull request closed without merging**